splitio/storage/pluggable.py

```diff
@@ -26,51 +27,78 @@ def __init__(self, pluggable_adapter, prefix=None):
-        :type split_name: str
+        :param feature_flag_name: Name of the feature to fetch.
+        :type feature_flag_name: str
 
         :rtype: splitio.models.splits.Split
         """
         try:
-            split = self._pluggable_adapter.get(self._prefix.format(split_name=split_name))
-            if not split:
+            feature_flag = self._pluggable_adapter.get(self._prefix.format(feature_flag_name=feature_flag_name))
+            if not feature_flag:
                 return None
-            return splits.from_raw(split)
+            return splits.from_raw(feature_flag)
         except Exception:
-            _LOGGER.error('Error getting split from storage')
+            _LOGGER.error('Error getting feature flag from storage')
             _LOGGER.debug('Error: ', exc_info=True)
             return None
 
-    def fetch_many(self, split_names):
+    def fetch_many(self, feature_flag_names):
         """
-        Retrieve splits.
+        Retrieve feature flags.
 
-        :param split_names: Names of the features to fetch.
-        :type split_name: list(str)
+        :param feature_flag_names: Names of the features to fetch.
+        :type feature_flag_name: list(str)
 
         :return: A dict with split objects parsed from queue.
         :rtype: dict(split_name, splitio.models.splits.Split)
         """
         try:
-            prefix_added = [self._prefix.format(split_name=split_name) for split_name in split_names]
-            return {split['name']: splits.from_raw(split) for split in self._pluggable_adapter.get_many(prefix_added)}
+            prefix_added = [self._prefix.format(feature_flag_name=feature_flag_name) for feature_flag_name in feature_flag_names]
+            return {feature_flag['name']: splits.from_raw(feature_flag) for feature_flag in self._pluggable_adapter.get_many(prefix_added)}
         except Exception:
-            _LOGGER.error('Error getting split from storage')
+            _LOGGER.error('Error getting feature flag from storage')
             _LOGGER.debug('Error: ', exc_info=True)
             return None
 
+    def get_feature_flags_by_sets(self, flag_sets):
+        """
+        Retrieve feature flags by flag set.
+
+        :param flag_set: Names of the flag set to fetch.
+        :type flag_set: str
+
+        :return: Feature flag names that are tagged with the flag set
+        :rtype: listt(str)
+        """
+        try:
+            sets_to_fetch = get_valid_flag_sets(flag_sets, self._config_flag_sets)
+            if sets_to_fetch == []:
+                return []
+
+            keys = [self._prefix(feature_flag_name) for feature_flag_name in sets_to_fetch]
+            result_sets = self._pluggable_adapter.get_many(keys)
+            return list(combine_valid_flag_sets(result_sets))
+        except Exception:
+            _LOGGER.error('Error fetching feature flag from storage')
+            _LOGGER.debug('Error: ', exc_info=True)
+            return None
```

Comment:
Should it return `[]` for consistency with line 91? Or is `None` used to represent that the function had an error?

Reply:
Yep, I added a None check from client class, thanks.